projects/playground/e2e/integration/playground.spec.ts

```diff
@@ -51,14 +45,13 @@ describe('Smoke tests', () => {
   it('do redirect login and show user and access token', () => {
     cy.visit('/');
     cy.get('#login').should('be.visible').click();
-
     cy.url().should('include', 'https://brucke.auth0.com/login');
     loginToAuth0();
-
     cy.get('[data-cy=userProfile]').contains(`"email": "${EMAIL}"`);
-
+    cy.get('[data-cy=idTokenClaims]').contains('__raw');
```

Comment:
Would it be worth checking something else? Like email ?

Reply:
We can do if you feel it would add value. I just check `__raw` as it only ever appears in the ID token claims. I just wanted a sanity check to make sure the value was being returned.

Reply:
I am fine with what we have. Was just curious if checking some value being present adds more value instead of checking the existence of a property. But let's keep it as is.